libs/movex/src/specs/MovexClientResourceObservable.spec.ts

```diff
@@ -138,7 +138,22 @@ describe('Observable', () => {
         expect(actual.val).toEqual('ChecksumMismatch');
       });
     });
+      // TODO: test xresource destroy which unsubscribes
+      test('Destroys the observable', async () => {
+        const xResource = new MovexResourceObservable(
+          'test-client',
+          rid,
+          counterReducer
+        );
+        xResource.setMasterSyncing(false);
+        const updateListener = jest.fn();
+        xResource.onUpdated(updateListener);
+        xResource.destroy();
+        await tillNextTick();
+        xResource.dispatch({
+          type: 'increment',
+        });
+        expect(updateListener).not.toHaveBeenCalled();
+      });
   });
```

Comment:
This looks good. One thing I would say, is to add an `expect` before the destroy as well, ensuring that it gets called.